app/models/nufs_account.rb

```diff
@@ -6,8 +6,13 @@ class NufsAccount < Account
 
   after_find :load_components
 
-  def set_expires_at!
+  def set_expires_at
     self.expires_at = ValidatorFactory.instance(account_number).latest_expiration
+    # If this Validator instance has an error, suppress it. The errors
+    # will be raised later.
+    rescue ValidatorError, AccountNumberFormatError
+      # Prevent the expires_at validation error message from appearing on the front end
+      self.expires_at ||= Time.current
```

Comment:
Is `Time.current` interchangeable with `Time.zone.now`? Asking because we use `Time.zone.now` in this project much more than `Time.current`.